builder/xenserver/xva/step_create_from_template.go

```diff
@@ -0,0 +1,191 @@
+package xva
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	"github.com/hashicorp/packer-plugin-sdk/packer"
+	xsclient "github.com/terra-farm/go-xen-api-client"
+	xscommon "github.com/xenserver/packer-builder-xenserver/builder/xenserver/common"
+)
+
+type stepCreateFromTemplate struct {
+	instance *xsclient.VMRef
+}
+
+func (self *stepCreateFromTemplate) Run(ctx context.Context, state multistep.StateBag) multistep.StepAction {
```

Comment:
This duplicates a significant amount of code from the iso stepCreateInstance struct. I think we should try to reduce this so that the code is easier to maintain if it ever needs to change.

I'll take a stab at doing this soon.